factory/consensus/consensusComponents.go

```diff
@@ -641,6 +655,19 @@ func (ccf *consensusComponentsFactory) createConsensusTopic(cc *consensusCompone
 	return ccf.networkComponents.NetworkMessenger().RegisterMessageProcessor(cc.consensusTopic, common.DefaultInterceptorsIdentifier, cc.worker)
 }
 
+func (ccf *consensusComponentsFactory) createPeerBlacklistHandler() (consensus.PeerBlacklistHandler, error) {
+	cache := timecache.NewTimeCache(defaultSpan)
```

Comment:
is there any conflict between the default span and the timespan added when blacklisting a peer?
when is the default span used?

Reply:
for this particular case, default span will not be actually used, since upsert is not using it